content/en/opentelemetry/agent/_index.md

```diff
@@ -7,9 +7,34 @@ private: true
   The Datadog Agent with embedded OpenTelemetry Collector is in private beta.
 {{< /callout >}} 
 
-## Guides
+## Overview
 
-{{< whatsnext desc=" " >}}
-    {{< nextlink href="/opentelemetry/agent/install_agent_with_collector" >}}Install the Datadog Agent with the OpenTelemetry Collector{{< /nextlink >}}
-    {{< nextlink href="/opentelemetry/agent/agent_with_custom_components" >}}Use Custom OpenTelemetry Components with Datadog Agent{{< /nextlink >}}
-{{< /whatsnext >}}
\ No newline at end of file
+The Datadog Agent with embedded OpenTelemetry Collector is an open source Collector distribution that combines the flexibility of OpenTelemetry with the comprehensive observability capabilities of Datadog. This integrated solution includes:
+
+- Built-in Datadog pipelines and extensions
+- Support for traces, metrics, and logs
+- A curated set of components for optimal performance with Datadog
+
+{{< img src="/opentelemetry/embedded_collector/architecture.png" alt="Architecture overview for Collector embedded in the Datadog Agent." style="width:100%;" >}}
+
+## Key benefits
+
+- **Data enrichment**: Receive out-of-the-box data enrichment with [universal service tagging][1] and [Kubernetes tag extraction][2], combined with [Container Monitoring][3] that leverages both OpenTelemetry-native data and Datadog's specialized data.
+- **Advanced data processing**: Access advanced data processing at the edge using OpenTelemetry (like tail-based sampling) with Datadog's native capabilities (such as [Sensitive Data Scanning][4]) for efficient and secure data handling.
+- **Flexibility with Datadog features**: Maintain OpenTelemetry compatibility accessing advanced Datadog products and features, including [Universal Service Monitoring][5] (USM) and [Network Performance Monitoring][7] (NPM).
+
+## Get started
```

Comment:
For this section suggest to reword as below: 

Whether you're new to Datadog or already familiar with OpenTelemetry (Otel), this guide will help you get started quickly and efficiently. Choose the guide below to find the right path for your needs.

1. (Quick start with the default Agent package)[https://docs-staging.datadoghq.com/brett0000FF/embedded-col-migration/opentelemetry/agent/install_agent_with_collector/]
For those starting fresh, using the Datadog Agent, or exploring OpenTelemetry for the first time, the default Agent package comes with an embedded Collector and a curated set of built-in Otel components designed to meet most needs right out of the box. This guide is a good fit for : 
- Starting from scratch and setting up monitoring without needing additional OTel components other than the built-in components [Link to list of built-in components].
- Using the Datadog Agent and testing OTel funcitonalities with build-in components [link to the list].
- Converging from Otel Collector to Datadog Agent, where no extra OTel components are required other than the build-in components [Link to list of built-in components]. 

2. Requiring additional OTel Components to be used with the Agent [Link to the doc]
If you require additional Otel components beyond what’s provided in the default package, we offer a flexible, bring-your-OTel-components workflow. This allows you to add any additional Otel components needed to extend the Datadog Agent's capabilities. 

3. Migrating from OpenTelemetry Collector to Datadog Agent
This is a comprehensive guide to help you migrate from an existing Otel Collector setup that may need additional OTe components beyond what's provided by default. Start by adding the necessary Otel components as described in Step 2, then proceed to migrate your configurations to Datadog. This guide is a good fit for:

- Transitioning from OTel Collector while preserving your existing setup and integrating it with Datadog.
- [optional] Customizing with additional components beyond the embedded options.
- Migrating your existing OTel configurations to maintain continuity.